src/status_im/ui/components/list_item/views.cljs

```diff
@@ -1,81 +1,378 @@
 (ns status-im.ui.components.list-item.views
-  (:require [status-im.ui.components.colors :as colors]
+  (:require [clojure.string :as string]
+            [reagent.core :as reagent]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.colors :as colors]
             [status-im.ui.components.icons.vector-icons :as icons]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.list-item.styles :as styles]
-            [status-im.utils.image :as utils.image]
             [status-im.ui.components.tooltip.views :as tooltip]
-            [status-im.ui.components.action-button.styles :as st]))
-
-; type - optional :default , :small
-
-; accessories - optional vector of :chevron, :check, :more or component or string
-
-; theme - optional :default, :wallet, :action, :action-red
-
-(defn list-item [{:keys [title subtitle accessories image image-path icon type theme on-press error content] :or {type :default theme :default}}]
-  (let [small? (= :small type)]
-    [react/touchable-highlight {:on-press on-press :disabled (not on-press)}
-     [react/view {:style (styles/container small?)}
-      ;;Image
-      (when image
-        [react/view {:margin-left 16}
-         (if (vector? image)
-           image
-           [image])])
-      ;;Icon
-      (when icon
-        [react/view {:margin-left 16}
-         [react/view (st/action-button-icon-container (when (= theme :action-red) colors/red-transparent-10))
-          [icons/icon icon (if (= theme :action-red) st/action-button-label-red st/action-button-label)]]])
-      (when image-path
-        [react/view {:margin-left 16}
-         [react/image {:source (utils.image/source image-path)
-                       :style  (styles/photo 40)}]])
-      ;;Title
-      (when title
-        [react/view {:style {:margin-left 16 :margin-right 16 :flex 1 :min-width 50}}
-         [react/text {:style           (merge (styles/title small? subtitle)
-                                              (when (= theme :action) st/action-button-label)
-                                              (when (= theme :action-red) st/action-button-label-red))
-                      :number-of-lines 1
-                      :ellipsize-mode  :tail}
-          title]
-         ;;Subtitle
-         (when subtitle
-           [react/text {:style           styles/subtitle
-                        :number-of-lines 1
+            [status-im.ui.screens.profile.db :as profile.db]))
+
+(def separator
```

Comment:
can we move it outside list-item? i believe it's used not only in list-item

Reply:
sure thing 👍
maybe there is one in an appropriate place already, do you know of it?
also probably better to call it `divider` as that is what @andmironov called it

Reply:
on a second thought, can we please keep it there?

Because `list-item` is intended to be used everywhere that is `list` like (which is pretty much everywhere). And `divider/separator` is a special kind of `item` in a list. In near future I was thinking `item-description`(see image below) could be one of the `item`.

![Profile -_ Primary (2)](https://user-images.githubusercontent.com/42058840/63843177-abc34a00-c9a5-11e9-9f19-8937cdd0484f.png)

Reply:
renamed to `divider` but did not move it in the light of explanation above. Let me know if it is still better to move it elsewhere and specify where